apps/fares/lib/fare_info.ex

```diff
@@ -181,18 +181,17 @@ defmodule Fares.FareInfo do
   end
 
   def mapper([
-        mode,
+        "express_bus",
         charlie_card_price,
         day_reduced_price,
         _month_reduced_price,
         _day_pass_price,
         _week_pass_price,
         month_pass_price | _
-      ])
-      when mode in ["inner_express_bus", "outer_express_bus"] do
+      ]) do
```

Comment:
It looks like this function's now nearly identical to 
```elixir
def mapper([
        "local_bus",
        charlie_card_price,
        day_reduced_price,
        _month_reduced_price,
        _day_pass_price,
        _week_pass_price,
        month_pass_price | _
      ])
``` 
now? So I guess we can consolidate them now (much like how `mode in ["inner_express_bus", "outer_express_bus"]` handled this before 🙈 ).

Reply:
Oh yeah!  I guess they all could have been consolidated all along.  I like it!